### PR TITLE
Make it easy to install ly on runit systems e.g. void

### PR DESCRIPTION
**Pull request by erkannt**
Copied nearly wholesale from https://github.com/drozdowsky/ly-void.

I only updated the Makefile and README to match what I could see for openrc.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -111,6 +111,25 @@ If you choose a tty that already has a login/getty running (has a basic login pr
 # rc-update del agetty.tty2
 ```
 
+### runit
+
+```sh
+# make
```
Comment (Cavernosa) [inline readme.md:117]: Why is there sh highlighting here? 
also `make` alone doesn't need root permissions

Comment (erkannt) [reply readme.md:117]: I was trying to follow what came above. `sh` highlighting just out of habit. Happy to remove it.

Would you prefer: 

```
make
make install installrunit
ln -s /etc/sv/ly /var/service/
```

or:

```
$ make
# make install installrunit
# ln -s /etc/sv/ly /var/service/
```

or:

```
make
sudo make install installrunit
sudo ln -s /etc/sv/ly /var/service/
```

I've got no preferences.

Comment (AnErrupTion) [reply readme.md:117]: Highlighting makes sense here, as other stuff above also has it.

Comment (AnErrupTion) [inline readme.md:117]: ``make`` itself doesn't require root, so please update it to the correct symbol ($)

**Review by AnErrupTion — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (Cavernosa) [reply readme.md:117]: > Would you prefer:

2nd one is what i meant. Just like AnErrupTion said in the change request below.

>Highlighting makes sense here, as other stuff above also has it.

What do you mean? No other code block has `sh` highlighting and `sh` highlight is not meant for CLI (as evident here, it's thinking the commands are comments).

Comment (erkannt) [reply readme.md]: Done.

**Pull request merged**